static/css/material.less

```diff
@@ -25,6 +25,9 @@
       padding-right: 10px;
       display: inline;
     }
+    .text {
```

Comment:
I don't think is sufficiently descriptive. I feel like the text should be within the h3 just styled to be small, rather than a separate element altogether. Semantically it's still the header even if it looks small. But this may be tricky to pull off and keep the correct layout.